clang/test/SemaCXX/cxx23-ctad-inherited-ctors.cpp

```diff
@@ -0,0 +1,260 @@
+// RUN: %clang_cc1 -fsyntax-only -std=c++23 -verify %s
+
```

Comment:
Can you add testcases for the template parameter packs? e.g. 

```
template <typename ...T> struct B {
  B(T...);
};
template <typename ...T> struct C : public B<T...> {
  using B<T...>::B;
};

C c(1);
```

Reply:
Thanks for catching this, I've fixed this case and added a test